### ISSUE TITLE
DVA memory warning

### ISSUE THREAD

**Issue by gitthangbaby** [007revad/Synology_HDD_db#136]
DVA3219 VM spam:
"Detected non-Synology recommended memory module configurations."

No matter what is set:
sed -i "s/support_memory_compatibility=\"no\"/support_memory_compatibility=\"yes\"/"  /etc.defaults/synoinfo.conf
sed -i "s/support_memory_compatibility=\"yes\"/support_memory_compatibility=\"no\"/"  /etc.defaults/synoinfo.conf
sed -i "s/support_memory_compatibility=\"no\"/support_memory_compatibility=\"yes\"/"  /etc/synoinfo.conf
sed -i "s/support_memory_compatibility=\"yes\"/support_memory_compatibility=\"no\"/"  /etc/synoinfo.conf

Here people believe opposite settings is good for DVA:
https://github.com/fbelavenuto/arpl/issues/307

_Another annoyance is nonstop nvidia warning: "NVRM: No NVIDIA graphics adapter found!" in dmesg_

Hope someone knows what's special about DVA.

**Comment by 007revad** [comment]
Your sed commands need to escape the " around the yes and no, otherwise sed fails to edit support_memory_compatibilty.
`sed -i "s/support_memory_compatibility="\no\"/support_memory_compatibility="\yes\"/" /etc/synoinfo.conf`

But there is an easier way to disable support_memory_compatibility:
```
synosetkeyvalue /etc/synoinfo.conf support_memory_compatibility no
synosetkeyvalue /etc.defaults/synoinfo.conf support_memory_compatibility no
```
Or to enable support_memory_compatibility:
```
synosetkeyvalue /etc/synoinfo.conf support_memory_compatibility yes
synosetkeyvalue /etc.defaults/synoinfo.conf support_memory_compatibility yes
```

And to check the values:
```
synogetkeyvalue /etc/synoinfo.conf support_memory_compatibility
synogetkeyvalue /etc.defaults/synoinfo.conf support_memory_compatibility
```

**Comment by gitthangbaby** [comment]
Sorry, the backslash was removed as I pasted it without code.
Still, in both ways the message stays there. By doing manually, by using -r switch, by using loader settings. No matter how it's set after boot.

**Comment by gitthangbaby** [comment]
Conclusion: -r sets 'no'. Reportedly, DVAs requires 'yes'. In my report, both don't work. Proposal that works:
/usr/lib/systemd/system/SynoMemCheck.service
`ExecStart=/bin/true`

**Comment by 007revad** [comment]
So you edited **/usr/lib/systemd/system/SynoMemCheck.service** to replace **ExecStart=/usr/syno/bin/syno_mem_check** with **ExecStart=/bin/true** ?

There's also a copy of SynoMemCheck.service in:
```
/usr/lib/systemd/system/basic.target.wants/SynoMemCheck.service
/usr/lib/systemd/system/syno-bootup-done.target.wants/SynoMemCheck.service
```

**Comment by gitthangbaby** [comment]
Correct, because i could not disable the service, it would come up like many other things in DSM. Looks like /bin/true is used in 10 other services already, quite popular.

**Comment by 007revad** [comment]
Fixed in [Release v3.1.63](https://github.com/007revad/Synology_HDD_db/releases/tag/v3.1.63)
